requirements.txt

```diff
@@ -6,3 +6,5 @@ pytest==7.4.3
 pytest-mock==3.12.0
 requests-mock==1.11.0
 coverage==7.3.2
+scikit-learn
```

Comment:
I am curious, what does scikit do?

Reply:
please disregard, now I see `sklearn` in the imports list

Reply:
I'm using sklearn's metric library to do the accuracy calculation/ confusion matrices